src/pages/login/CoreConnect.tsx

```diff
@@ -55,10 +55,11 @@ const CoreConnect = () => {
           setPathname('/login/user/select');
         }
         actions.setSubmitting(false);
+        actions.setStatus(null);
       })
       .catch((err) => {
         const errorMessages = errorToString(err);
-        actions.setErrors({ address: errorMessages }); //TODO: put the error in a better place, it's not just an address problem
+        actions.setStatus({ error: errorMessages });
```

Comment:
doesn't really have to be an object?
but this is fine

actually we should probably do the same for the other errors (setting status to an object, with the key "error" being the error message)